.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Standard Angular App Flow
+
+on:
+  pull_request:
+    branches: [ "main", "development" ]
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'  # Adjust this to the Node.js version your project needs
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run linting
+        run: npm run lint
```

Comment:
In package.json update script to, we don't want to have warnings too.

"lint": "ng lint --max-warnings=0",

Reply:
updated this config with the latest PR at #10